docs/introduction.md

```diff
@@ -67,83 +96,207 @@ Flow is based upon the following set of rules:
-They are responsible for scaling the computational power of the blockchain, and are the only node role to have access to the **execution state**.
-They produce cryptographic attestations declaring the result of their efforts in the form of **execution receipts**.
-The receipts can be used to challenge the claims of an execution node when they are shown to be incorrect.
-They are also used to create proofs of the current state of the chain once they are known to be correct.
-In order to allow [verification nodes](#verification-nodes) to process the execution receipts, computations of a block are split into chunks.
-Each execution node publishes additional information about each chunk in its execution receipt for the block executed.
-Each chunk corresponds to a [collection](#collection-nodes).
-
-Errors introduced by the execution nodes are always guaranteed to have four critical attributes:
+Flow is designed to guarantee that any error introduced by the execution nodes are always guaranteed to have four critical attributes:
 
 * **Detectable**: A deterministic process has an objectively correct output. Therefore, even a single honest node in the network can detect deterministic faults and prove the error to all other honest nodes by pointing out the part of the process that was executed incorrectly.
 * **Attributable**: The output of all deterministic processes in Flow but be signed with the identity of the node that generated those results. As such, any error that has been detected can be clearly attributed to the node(s) that were responsible for that process.
 * **Punishable**: All nodes participating in a Flow network, including execution nodes, must put up a [stake](#staking) that can be [slashed](#slashing) if they are found to have exhibited [Byzantine behavior](#byzantine-fault). Since all errors in deterministic processes are detectable and attributable, those errors can be reliably punished via [slashing](#slashing).
 * **Recoverable**: The system must have a means to undo errors are they are detected. The property services to deter malicious actors from inducing errors that benefit them more than the slashing penalty.
 
-##### Consensus Nodes
+### Execution State vs Protocol State
+
+Flow differentiates between two sorts of states:
+1. Execution state (computational state)
+2. Protocol state (network infrastructure state)
+
+Execution and protocol states are maintained and updated independently. Execution state is maintained and updated by the execution nodes, while the protocol state is maintained and updated by the consensus nodes.
+
+The execution state contains the register values which are modified by the transaction execution. Updates to the execution states are done by the execution nodes but their integrity is governed by the verification process.
+
+The protocol state keeps track of the system related features like staked nodes, node roles, public keys, network addresses and staking amounts.
```

Comment:
The protocol state also tracks the chain of blocks by creating consensus amongst network participants on what the objectiv pre-determined order of transactions is (which is done implicitly by ordering collections and a deterministic algorithm to then determine the transaction order within the blocks).